<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InterfaceOptionsWidget</class>
 <widget class="QWidget" name="InterfaceOptionsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>349</width>
    <height>261</height>
   </rect>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Cursor movement after pressing Enter:</string>
     </property>
     <property name="buddy">
      <cstring>m_cursorMovement</cstring>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="KComboBox" name="m_cursorMovement">
     <property name="whatsThis">
      <string>When you have selected a cell, pressing the Enter key will move the cell cursor one cell left, right, up or down, as determined by this setting.</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Func&amp;tion shown in status bar:</string>
     </property>
     <property name="buddy">
      <cstring>m_statusBarFunction</cstring>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="KComboBox" name="m_statusBarFunction">
     <property name="whatsThis">
      <string>This drop down selection box can be used to choose the calculation performed by the Statusbar Summary  function.</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_6">
     <property name="text">
      <string>&amp;Unit:</string>
     </property>
     <property name="buddy">
      <cstring>m_unit</cstring>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="KComboBox" name="m_unit">
     <property name="whatsThis">
      <string>Choose the default unit that will be used in your sheets.</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="m_indentationLabel">
     <property name="text">
      <string>I&amp;ndentation step:</string>
     </property>
     <property name="buddy">
      <cstring>m_indentationStep</cstring>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="KoUnitDoubleSpinBox" name="m_indentationStep">
     <property name="whatsThis">
      <string>Lets you define the amount of indenting used by the Increase Indent and Decrease Indent actions.</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="2">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Colors</string>
     </property>
     <layout class="QFormLayout" name="formLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>&amp;Grid:</string>
        </property>
        <property name="buddy">
         <cstring>m_gridColor</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="KColorCombo" name="m_gridColor">
        <property name="whatsThis">
         <string>Click here to change the grid color ie the color of the borders of each cell.</string>
        </property>
        <item>
         <property name="text">
          <string>Custom...</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string/>
         </property>
        </item>
        <item>
         <property name="text">
          <string/>
         </property>
        </item>
        <item>
         <property name="text">
          <string/>
         </property>
        </item>
        <item>
         <property name="text">
          <string/>
         </property>
        </item>
        <item>
         <property name="text">
          <string/>
         </property>
        </item>
        <item>
         <property name="text">
          <string/>
         </property>
        </item>
        <item>
         <property name="text">
          <string/>
         </property>
        </item>
        <item>
         <property name="text">
          <string/>
         </property>
        </item>
        <item>
         <property name="text">
          <string/>
         </property>
        </item>
        <item>
         <property name="text">
          <string/>
         </property>
        </item>
        <item>
         <property name="text">
          <string/>
         </property>
        </item>
        <item>
         <property name="text">
          <string/>
         </property>
        </item>
        <item>
         <property name="text">
          <string/>
         </property>
        </item>
        <item>
         <property name="text">
          <string/>
         </property>
        </item>
        <item>
         <property name="text">
          <string/>
         </property>
        </item>
        <item>
         <property name="text">
          <string/>
         </property>
        </item>
        <item>
         <property name="text">
          <string/>
         </property>
        </item>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Page Out&amp;line:</string>
        </property>
        <property name="buddy">
         <cstring>m_pageOutlineColor</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="KColorCombo" name="m_pageOutlineColor">
        <property name="whatsThis">
         <string>When the View -&gt;Show Page Borders menu item is checked, the page borders are displayed. Click here to choose another color for the borders than the default red.</string>
        </property>
        <item>
         <property name="text">
          <string>Custom...</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string/>
         </property>
        </item>
        <item>
         <property name="text">
          <string/>
         </property>
        </item>
        <item>
         <property name="text">
          <string/>
         </property>
        </item>
        <item>
         <property name="text">
          <string/>
         </property>
        </item>
        <item>
         <property name="text">
          <string/>
         </property>
        </item>
        <item>
         <property name="text">
          <string/>
         </property>
        </item>
        <item>
         <property name="text">
          <string/>
         </property>
        </item>
        <item>
         <property name="text">
          <string/>
         </property>
        </item>
        <item>
         <property name="text">
          <string/>
         </property>
        </item>
        <item>
         <property name="text">
          <string/>
         </property>
        </item>
        <item>
         <property name="text">
          <string/>
         </property>
        </item>
        <item>
         <property name="text">
          <string/>
         </property>
        </item>
        <item>
         <property name="text">
          <string/>
         </property>
        </item>
        <item>
         <property name="text">
          <string/>
         </property>
        </item>
        <item>
         <property name="text">
          <string/>
         </property>
        </item>
        <item>
         <property name="text">
          <string/>
         </property>
        </item>
        <item>
         <property name="text">
          <string/>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="5" column="0" colspan="2">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>318</width>
       <height>16</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KComboBox</class>
   <extends>QComboBox</extends>
   <header>kcombobox.h</header>
  </customwidget>
  <customwidget>
   <class>KColorCombo</class>
   <extends>QComboBox</extends>
   <header>kcolorcombo.h</header>
  </customwidget>
  <customwidget>
   <class>KoUnitDoubleSpinBox</class>
   <extends>QDoubleSpinBox</extends>
   <header>KoUnitDoubleSpinBox.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
